src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,7 +26,35 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         if (!isValidAddress(trimmedAddress)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
-        this.value = trimmedAddress;
+        String[] splitFormOfAddress = trimmedAddress.split(",");
+        Block blockAddress ;
```

Comment:
It is good these are not `public`.
However, try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.